universum/modules/reporter.py

```diff
@@ -151,8 +156,9 @@ def _report_steps_recursively(self, block: Block, text: str, indent: str) -> Tup
             text += indent + str(block) + '\n'
             self.out.log_summary_step(indent + block_title, has_children, block.status)
         elif not block.is_successful():
-            text += str(block) + '\n'
-            self.out.log_summary_step(block_title, has_children, block.status)
+            if not self.settings.only_fails_short or not block.children:
```

Comment:
There is already `has_children` variable defined within this function

Reply:
tnx